src/Czechia/README.md

```diff
@@ -61,45 +98,9 @@ if (response.IsSuccess)
 - More examples are presented [here](doc/examples.md).
 - Some of the data items are explained [here](doc/data.md).
 
-# NuGet
-
-We have published the library as [Mews.Fiscalizations.Czechia](https://www.nuget.org/packages/Mews.Fiscalizations.Czechia/).
-
-Alternatively, the NuGet package can be generated by running the following command:
-
-```
-src\Mews.Fiscalizations.Czechia> nuget pack .\Mews.Fiscalizations.Czechia.csproj -IncludeReferencedProjects
-```
-
-# Tests
-Test are not currently running on Travis CI, as their network infrastructure is blocking HTTPS traffic directed into Europe, where the EET servers are located.
-
-Otherwise, a xUnit test suite is a part of the library, it contains end-to-end tests that communicates with Playground EET servers to verify the lib is really working against the current EET version.
-
-# Authors
+## 🧑 Authors
 Development: [@jirihelmich](https://github.com/jirihelmich)
 Code review: [@siroky](https://github.com/siroky), [@onashackem](https://github.com/onashackem)
 
-# Who uses the library in production?
-- [Mews](https://mews.com) - Property Management Solution for the 21st century.
-- [Hlídač EET](http://hlidaceet.cz) - Watchdog of the EET API endpoint and related services.
-- Keszh, Neszh - [Hart](http://hartphp.com.pl/) Internal ERP System
-- [Ankerdata EasiPOS](http://easipos.ankerdata.com/) - POS system
-- [sySOft](http://www.sysoft.cz/) - POS system
-
-We would like to hear your story and know who users of the lib are. Please, thank us for providing the library by sharing with us, who you are and letting us add you into this list.
-
-The time to implement this was kindly provided by [Mews](https://mews.com).
-
-# Donate
-There is no need to donate the project, but thanks for considering it! Instead, if you like the project, star it here on GitHub :-)! Thanks!
-
-If you still insist on donating, we accept gummy bears at Mews Systems s.r.o., Náměstí IP Pavlova 5, Vinohrady 120 00 Prague. This project was, of course, powered by a huge pile of gummy bears ;-)
-
-# Related projects
-- [eet-client](https://github.com/todvora/eet-client) by [@todvora](https://github.com/todvora): Java, MIT
-- [SwiftEET](https://github.com/charlieMonroe/SwiftEET) by [@charlieMonroe](https://github.com/charlieMonroe): Swift, GPL-3.0
-
-# Credits
-- [Komodosoft](http://www.komodosoft.net) For publishing [the post](http://www.komodosoft.net/post/2016/03/24/sign-a-soap-message-using-x-509-certificate.aspx) about signing a SOAP message without using WCF.
```

Comment:
No credits to Komodosoft? 
No related projects?

Reply:
Komodosoft website is down, i already checked 3 days in row.

Reply:
related project
eet-client last updated: 17 months ago
SwiftEET: last updated 4 years ago.

Reply:
Fair enough